### ISSUE TITLE
Debugging fresh clone gives "RunCommand property is not defined"

### ISSUE THREAD

**Issue by JVimes** [MahApps/MahApps.Metro#3245]
I can successfully build the latest develop branch [commit](https://github.com/MahApps/MahApps.Metro/commit/d99b032a82cc5c52ae48642c6f110ac74ae48062), but when I try to debug I get the following error (manually running `.paket\paket restore` made no difference).

![image](https://user-images.githubusercontent.com/6148201/39898686-17b30536-5475-11e8-851a-786b7eab378b.png)

## What steps will reproduce this issue?

1. Do a fresh clone of https://github.com/MahApps/MahApps.Metro.git

1. Open src\MahApps.Metro.sln. Wait 1min for it to load. Output is this:

	```
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net45\Themes\Dialogs\InputDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net45\Themes\Dialogs\MessageDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net45\Themes\Dialogs\MessageDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net45\Themes\Dialogs\LoginDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net45\Themes\Dialogs\ProgressDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net45\Themes\Dialogs\ProgressDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net45\Themes\Dialogs\LoginDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net45\Themes\Dialogs\InputDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net47\Themes\Dialogs\ProgressDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net47\Themes\Dialogs\ProgressDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net47\Themes\Dialogs\MessageDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net47\Themes\Dialogs\MessageDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net40\Themes\Dialogs\LoginDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net40\Themes\Dialogs\ProgressDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net40\Themes\Dialogs\ProgressDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net40\Themes\Dialogs\MessageDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net40\Themes\Dialogs\InputDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net40\Themes\Dialogs\LoginDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net40\Themes\Dialogs\InputDialog.g.cs'.
	[Failure] Could not find file 'C:\Users\johnb\Desktop\MahApps.Metro\src\MahApps.Metro\obj\Debug\net40\Themes\Dialogs\MessageDialog.g.cs'.
	```

1. Select MahApps.Metro.Demo as the startup project.

1. Build Solution (F6). I get "build succeeded". Output is [this](https://gist.githubusercontent.com/JVimes/fd70f87404a48f0a3e104ed020afdac7/raw/7a0f85ee4a719bf2bd1ae553e9832cc003e788a4/gistfile1.txt).

1. Start Debugging (F5). I get, `Unable to run your project. The "RunCommand" property is not defined.`

1. I tried `.paket\paket restore`, no change.

### Environment

- MahApps.Metro __Latest develop branch, also happend with v1.6.2__
- Windows OS __10__
- Visual Studio __2017 Community (15.7.1)__
- .NET Framework __4.0, 4.5, 4.6, 4.7__

**Comment by punker76** [comment]
@JVimes This sounds like a bug with the new msbuild sdk together with VS and WPF projects...

**Comment by JVimes** [comment]
Okay, if I get a minute, I'll see if I can narrow it down.

**Comment by punker76** [comment]
@JVimes Is this still an issue for you?

**Comment by JVimes** [comment]
@punker76 It's still an issue. Unfortunately I haven't had time to investigate. I'm up to v15.7.3 of Visual Studio. Let me know if there's anything you'd like me to try.

**Comment by punker76** [comment]
@JVimes Any chance to test it again with the latest dev source and these hints https://github.com/MahApps/MahApps.Metro/wiki/Building-the-MahApps.Metro-solution ?

**Comment by JVimes** [comment]
Thanks, @punker76, I got it working. I was missing the .NET Core SDK and the .NET Framework 4.7 SDK.

Also, some order of operations lead it to link v3.1.0 of Costura.Fody, which gave build error, "Method not found Fody.TypeSystem Fody.BaseModuleWeaver.get_TypeSystem()". I cleared my NuGet caches and now it links Costura.Fody v2.0.0 and builds.

I ran into other quirks (errors when running .paket\paket restore before building in Visual Studio, having to build twice on a clean repo, one other I forget), but now I can't repro them. I wonder if clearing NuGet caches does good things for the MahApps.Metro build. Thanks again.
